packages/webapp/pages/devcard.tsx

```diff
@@ -347,59 +396,18 @@ const seo: NextSeoProps = {
 };
 
 const DevCardPage = (): ReactElement => {
-  const { user } = useContext(AuthContext);
-  const queryClient = useQueryClient();
-  const [isLoadingImage, setIsLoadingImage] = useState(false);
-  const [devCardSrc, setDevCardSrc] = useState<string>();
-  const [imageError, setImageError] = useState<string>();
   const { completeAction, checkHasCompleted } = useActions();
+  const [shouldShowCard, setShouldShowCard] = useState(false);
   const isDevCardGenerated = checkHasCompleted(ActionType.DevCardGenerate);
 
-  const onError = () => {
-    setImageError(labels.error.generic);
-    setIsLoadingImage(false);
-  };
-
-  const { mutateAsync: generateDevCard } = useMutation(
-    ({ theme, type, isProfileCover, showBorder }: GenerateDevCardParams = {}) =>
-      request(graphqlUrl, GENERATE_DEVCARD_MUTATION, {
-        theme,
-        type,
-        isProfileCover,
-        showBorder,
-      }),
-    {
-      onMutate({ theme, isProfileCover, showBorder }) {
-        const devCardQueryKey = generateQueryKey(RequestKey.DevCard, {
-          id: user.id,
-        });
-
-        queryClient.setQueryData(devCardQueryKey, (oldData: DevCardData) => {
-          if (oldData) {
-            return { ...oldData, theme, showBorder, isProfileCover };
-          }
-        });
-      },
-      onSuccess({ devCard }) {
-        const img = new Image();
-        const { imageUrl } = devCard;
-        img.src = imageUrl;
-        img.onload = () => {
-          setDevCardSrc(imageUrl);
-          setIsLoadingImage(false);
-          completeAction(ActionType.DevCardGenerate);
-        };
-        img.onerror = onError;
-      },
-      onError,
-    },
-  );
-
   const stepProps: StepProps = {
-    onGenerateImage: generateDevCard,
-    devCardSrc,
-    isLoadingImage,
-    error: imageError,
+    onGenerateImage: () => {
+      setShouldShowCard(true);
```

Comment:
Why are we using both these checks?

Reply:
The first one is just to switch to the other screen since as per design we have this first "Generate devcard" screen.

Reply:
The second one, I can't quite remember but I believe it is supposed to be some sort of highlighting pulse to some element relate

Reply:
After checking the codebase, it seems we only use it here, this would make the screen stop showing the first page once the user has seen it already.

Reply:
We might actually not need the state in this case.

Reply:
Yup, the isDevCardGenerated is related to the action we set when user has seen the generate page, so I don't think we need `shouldShowCard` this state. Let's remove